pandas/tests/libs/test_hashtable.py

```diff
@@ -272,6 +272,14 @@ def test_value_count(self, dtype, type_suffix, writable):
         tm.assert_numpy_array_equal(np.sort(keys), expected)
         assert np.all(counts == 5)
 
+    def test_value_count_stable(self, dtype, type_suffix, writable):
+        value_count = get_ht_function("value_count", type_suffix)
```

Comment:
can you add the issue number here as a comment

Reply:
Done https://github.com/pandas-dev/pandas/pull/39009/commits/de88c5458c78cc65e5371d60495ecd09dba3b117